macros/ui/niceTables.pl

```diff
@@ -1432,6 +1440,16 @@ sub getLaTeXthickness {
 	return $output;
 }
 
+sub getLaTeXcolumnWidth {
+	my $input = shift;
+	return '' unless $input;
+	if ($input =~ /^\d+$/) {
```

Comment:
The rest of this sub can be written a little more compactly with:
`return ($input =~ /^\d+$/) ? '|' x $input : "!{\\vrule width $input}";`